VK/VK/Animations/Session/Settings.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Settings.swift
+//  VK
+//
+//  Created by Алена Панченко on 21.11.2022.
+//
+
+import UIKit
+
+/// Хранение данных о текущей сессии
+final class Settings {
```

Comment:
```suggestion
final class Session {
```
По названию я ожидаю тут настройки, по описанию хранение